### ISSUE TITLE
breaking dependencies

### ISSUE THREAD

**Issue by etconsilium** [pfalcon/picotui#34]
picotui needs termios
```
>python example_dialogs.py
  File "...tty.py", line 5, in <module> from termios import *
ModuleNotFoundError: No module named 'termios'
```
but 
```
>pipenv install pycopy-termios
Installing pycopy-termios…
Adding pycopy-termios to Pipfile's [packages]…
Installation Succeeded
Locking [packages] dependencies…
Locking Failed!
...
pipenv.patched.notpip._internal.exceptions.
Installation Error: Command "python setup.py egg_info" failed with error code 1 ...
```

мне кажется, вы зря массово удалили setup.py https://github.com/pfalcon/pycopy-lib/commit/4e9a8ad59805f861b0c018edcf0abb4ae140b691
отсутствие сетапа ломает автоматическую установку по зависимостям

**Comment by pfalcon** [comment]
I'm not sure what issue is reported here, it seems that a number of things mixed up here.

~~~
>python example_dialogs.py
  File "...tty.py", line 5, in <module> from termios import *
ModuleNotFoundError: No module named 'termios'
~~~

Impossible. "termios" is a standard module of (C)Python: https://docs.python.org/3.7/library/termios.html, so you likely have broken CPython installation.

Anyway, while it's unclear what you report, let me try to explain the situation:

1. picotui codebase is intended to be compatible with both CPython and Pycopy.
2. However, at this time, on PyPI, only package for CPython is provided.
3. Pycopy's `upip` can't install CPython's packages (e.g. picotui package provided on PyPI).
4. CPython's `pip` can't install Pycopy packages.

Summing up, if you're interested in using picotui with Pycopy, you'd need to use it from git clone, not installed via `upip`.

https://github.com/pfalcon/pycopy-lib/commit/4e9a8ad59805f861b0c018edcf0abb4ae140b691 has nothing to do with anything of that still, setup.py are autogenerated files, and that's why they're removed from *git repository*, because they're noise to track. They're still there (autogenerated!) at the package creation time, and explicitly removed from `upip` packages, as `upip` never uses them.

**Comment by pfalcon** [comment]
And yeah, it would be nice to get the same package to be installable both using `pip` and `upip`, and that's ongoing task to figure out the best way to do that, that carry it out.
